packages/config/config/devices/0x015f/mh9-co2-wd_2.4.json

```
@@ -0,0 +1,60 @@
+{
+	"manufacturer": "McoHome Technology Co., Ltd.",
+	"manufacturerId": "0x015f",
+	"label": "MH9-CO2-WD",
+	"description": "CO2 Monitor Air quality detector",
+	"devices": [
+		{
+			"productType": "0x0902",
+			"productId": "0x5102"
+		}
+	],
+	"firmwareVersion": {
+		"min": "2.4",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"1": {
+			"label": "CO2 Notification Threshold",
+			"description": "CO2 Notification threshold 1 to 5000ppm",
+			"unit": "ppm",
+			"valueSize": 2,
+			"minValue": 1,
+			"maxValue": 5000,
+			"defaultValue": 1000
+		},
+		"2": {
+			"label": "CO2 reporting threshold",
+			"description": "0 report disabled, 1~127: report when change >= n*5ppm, -1~-127: report when change >= (n+256)*5ppm, 10 by default, 10*5ppm=50ppm",
+			"valueSize": 1,
+			"minValue": -127,
+			"maxValue": 127,
```

This might not be obvious, but this ultra complicated description is not necessary when defining the parameter as `unsigned`. This is equivalent:

```suggestion
			"valueSize": 1,
			"unit": "5ppm",
			"unsigned": true,
			"minValue": 0,
			"maxValue": 255,
			"options": [
				{
					"label: "Disable",
					"value": 0
				}
			]
```

Please change parameter 3 and 4 accordingly.